### PR TITLE
Branch guest model

### PR DESCRIPTION
**Pull request by weibinlim**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Rsvp.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+     *
+     * @param rsvp A valid RSVP status.
+     * @throws NullPointerException if the given rsvp is null.
+     * @throws IllegalArgumentException if the given rsvp is invalid.
+     */
+    public Rsvp(String rsvp) {
+        requireNonNull(rsvp);
+        checkArgument(isValidRsvp(rsvp), MESSAGE_CONSTRAINTS);
+        this.rsvp = rsvp;
+    }
```
Comment (slye20) [inline src/main/java/seedu/address/model/person/Rsvp.java:38]: It looks good, but just a small thing would be maybe to standaridise how rsvp is being stored. Either all upper or lower cases so it looks the same when toString is called.

**Comment by YangQF2002** [comment]
lgtm! (other than what sam mentioned)

File: src/main/java/seedu/address/model/person/Rsvp.java (hunk 1 above)
Comment (hovanleong) [reply src/main/java/seedu/address/model/person/Rsvp.java:38]: Agreed, can call toUpperCase() perhaps.

**Comment by YangQF2002** [comment]
lgtm

**Comment by slye20** [comment]
lgtm

**Pull request merged**